node/src/chain_spec/dev.rs

```diff
@@ -13,16 +15,34 @@ pub fn dev_config(
         ChainType::Local,
         move || {
             generic_genesis(
+                #[cfg(feature = "parachain")]
+                AdditionalChainSpec {
+                    inflation_info: crate::chain_spec::DEFAULT_COLLATOR_INFLATION_INFO,
+                    stakers: vec![
+                        (
+                            get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+                            None,
+                            1_000 * zeitgeist_primitives::constants::BASE,
+                        ),
+                        (
+                            get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+                            None,
+                            1_000 * zeitgeist_primitives::constants::BASE,
+                        ),
```

Comment:
Do we need Bob as a staker here? below Bob is not set as an authority

Reply:
Aura and Grandpa authorities are for standalone nodes and these `Alice//stash` and `Bob//stash` accounts are for an initial set of stakers for testing.

Reply:
🤦 d'oh... you're right